src/pip/_internal/cli/cmdoptions.py

```diff
@@ -302,7 +302,8 @@ def exists_action():
     dest='cert',
     type='path',
     metavar='path',
-    help="Path to alternate CA bundle.",
+    help="Path to PEM-encoded CA certificate bundle. "
+         "If provided, overrides the default.",
```

Comment:
@chrahunt proposes in https://github.com/pypa/pip/issues/6720#issuecomment-538791684 have this link back to SSL Certificate Verification section - how?

Reply:
IMO it’s enough to say something like *See “SSL Certificate Verification” in pip documentation for more information*.